doc/rest.md

```diff
@@ -226,12 +227,13 @@ Body parameters:
       - file names, i.e. array of strings
    - if file names are provided, files must be uploaded subsequently using [POST Layer Chunk](#post-layer-chunk)
    - if published file has empty bounding box (i.e. no features), its bounding box on WMS/WFS endpoint is set to the whole World
+   - if QML style is used (either directly within this request, ir indirectly from previous state on server), it must list all attributes contained in given data file
```

Comment:
=> 'if QML style is used (either directly within this request, **or** indirectly from previous state on server), it must list all attributes contained in given data file'